app/models/team.rb

```diff
@@ -14,6 +14,13 @@
 class Team < ApplicationRecord
   has_many :user_teams, dependent: :destroy
   has_many :users, through: :user_teams
+  before_validation :strip_name
 
   validates :name, presence: true, length: { maximum: 100 }, uniqueness: true
+
+  private
+
+  def strip_name
+    self.name = name.strip unless name.nil?
```

Comment:
I believe this would work as well with less code: `name&.strip!` (also for `strip_first_name_last_name`)

Reply:
Yes, you're correct!